openreview/arr/management/setup_reviewer_matching.py

```diff
@@ -233,6 +234,31 @@ def replace_edge(existing_edge=None, edge_inv=None, new_weight=None, submission_
                 members=[]
             )
         )
+        
+        # Create Reviewers/Previous group for resubmissions
+        if submission.id in resubmission_ids:
+            reviewers_previous_id = venue.get_reviewers_id(number=submission.number) + '/Previous'
```

Comment:
instead of using a new group, can you use the group of the previous submission from a previous cycle?

Reply:
When authors opt-in for an anonymous pre-print, so to make their submission public, they set `preprint` to `True`. There's an invitation where the dateprocess posts edits using the `Preprint_Release_Submission` to submissions with `preprint: True` that sets the readers of the note to `everyone` and sets readers for all the fields that need to be hidden from the public